package.json

```diff
@@ -21,7 +21,7 @@
   "license": "MIT",
   "repository": "git@github.com:CatchRelease/arbor.git",
   "engines": {
-    "node": "~10"
+    "node": "~8"
```

Comment:
What are your thoughts on `>=8.0.0 <11.0.0` since we know arbor works well with 10 and the reason we are lowering it here is to use it with C&R?

Reply:
You're right. I forgot how `~` works. I was thinking it would go up from there but actually it would do any minor versions for 8.

I don't think we necessarily need to do less than 11.0 but rather less than 11.0, but rather 11.11.0 since 11.11 is where we started seeing issues.